src/lib.rs

```diff
@@ -27,7 +27,10 @@ use core::arch::global_asm;
 use core::arch::naked_asm;
 pub use entry::secondary_entry;
 #[cfg(feature = "initial-pagetable")]
-pub use pagetable::{DEFAULT_MAIR, DEFAULT_SCTLR, DEFAULT_TCR, InitialPagetable};
+pub use pagetable::{
+    __enable_mmu_el1, __enable_mmu_el2, __enable_mmu_el3, DEFAULT_MAIR, DEFAULT_SCTLR, DEFAULT_TCR,
```

Comment:
To avoid cluttering users' IDE autosuggestions, I'd suggest placing `__enable_mmu_elX` functions in a separate `__private` module, similarly to some popular crates (see https://github.com/serde-rs/serde/blob/e42684f9a773f0bd0d85f293f0a9034c1ba6c984/serde/build.rs#L7-L27 or https://github.com/nvzqz/divan/blob/bca5c9676a35751d0a8164df7d79bda70f23286b/src/lib.rs#L16-L19 for reference)